README.md

```diff
@@ -2,3 +2,39 @@
 
 [![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by/4.0/)
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+
+#Introduction to Project
+   This project will analyze a large dataset which characterizes the life histories of many mammalian taxa for potential trends. The purpose of this project is to determine if potential relationships between life history characteristics of mammals hold across a wide range of taxa. We want to undertake this project due to a shared interest in the evolution and ecology of animals. 
+  
+#Description of Data
+  This dataset brings together extensive data on the life histories of a wide range of mammalian taxa for the purpose of providing the opportunity for large-scale comparative life history analyses. All data in this dataset were pulled from published literature and compiled. We will likely use the variables: weaning age, weaning size, litter size, and litters per year. We will analyze these variables across a wide range of taxa (likely at the order level) to determine if there are correlations between them.
+
+Morgan Ernest SK. 2003. Data from: Life history characteristics of placental non-volant mammals. *Ecology* 84: 3402.  
+  (http://www.esapubs.org/archive/ecol/E084/093/default.htm#data)
+  
+#Team Description
+
+Carter
+EEB Specialist - 4th year
+Research interests - herpetology, ecology 
+Group responsibilities - writing
```

Comment:
To be rendered properly, this should be a list, like so:

```
- Carter
    - EEB specialist - 4th year
    - Research ... etc
```

Done to all of them (all members)